docs/info/long-range.md

```diff
@@ -1,15 +1,15 @@
 ---
 template: main.html
-description: ExpressLRS boasts great range at any given packet rate. Here's what others reached with their aircrafts.
+description: ExpressLRS boasts a great range at any given packet rate. Here's what others reached with their aircraft.
 ---
 
 ![FAQ-Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-hardware/master/img/community.png)
 
-One of the most frequently asked questions that gets asked by people who are interested in, but haven't yet tried ELRS is "How far does it go, and at what power level?
+One of the most frequently asked questions that are asked by people who are interested in, but haven't yet tried ELRS is "How far does it go, and at what power level?
 
-The following table is a leaderboard of the current record holder for each packet rate, and the longest distance from home. Note that not every flight resulted in a failsafe at max range, so the link may go (much) futher in some cases. The leaderboard is grouped into 2 tables, each with 2 sub tables. The 2 tables are the frequency of the link used (either 2.4GHz or 900MHz), subdivided into two power levels (50mw or less, 100mW+). This aims to give a sense of how well ELRS performs even at lower power and also demonstrate how far it can be pushed out at higher power levels. Each table will have a maximum of 20 entries, so get up there!
+The following table is a leaderboard of the current record holder for each packet rate, and the longest distance from home. Note that not every flight resulted in a failsafe at max range, so the link may go (much) further in some cases. The leaderboard is grouped into 2 tables, each with 2 sub-tables. The 2 tables are the frequency of the link used (either 2.4GHz or 900MHz), subdivided into two power levels (50mw or less, 100mW+). This aims to give a sense of how well ELRS performs even at lower power and also demonstrate how far it can be pushed out at higher power levels. Each table will have a maximum of 20 entries, so get up there!
```

Comment:
```suggestion
 Note that not every flight resulted in a failsafe at max range, so the link may go (much) further in some cases. 
```
 Note that not every flight resulted in a failsafe at max range, the link could go further in some cases.

I think this sentence would be better? what's your opinion?